helmclient.go

```diff
@@ -394,6 +395,47 @@ func (c *Client) InstallFromTarball(path, ns string, options ...helmclient.Insta
 	return nil
 }
 
+// RunReleaseTest runs the tests for a Helm Release. The releaseName is the
+// name of the Helm Release that is set when the Helm Chart is installed. This
+// is the same action as running the helm test command.
+func (c *Client) RunReleaseTest(releaseName string, options ...helmclient.ReleaseTestOption) error {
+	c.logger.Log("level", "debug", "message", fmt.Sprintf("running tests for release '%s'", releaseName))
```

Comment:
Little nit: `running release tests for ...` seems more intuitive to me. Not sure though, I feel the naming in this whole helm/appr world is not very clear to me.

Reply:
Thanks that's nicer I'll update. Yes helm and appr both having releases is a pain. Especially as it comes out of essentially the same community.